en/getting_started/basic_flight_modes.md

```diff
@@ -2,185 +2,178 @@
-You can read in depth about Flight Modes in the Flight Mode Setup page.
+#### Manual/Stabilized Mode
 
+In Stabilized flight your copter will hover in level flight once the Pitch and Roll sticks are at center position. You will have a full manual angle control over the copter. If your craft is perfectly set and balanced it will hold the altitude if throttle is at 50% in the middle of the scale at calm weather. If there is a wind blowing the craft will drift in the direction of the wind and you have to control the throttle all the time during wind gusts to hold altitude. In order to perform a maneuver, you have to hold your Pitch, Roll and Yaw command, otherwise the craft will level itself the moment you release the control sticks.
 
+![MC Manual Flight](../../images/flight_modes/manual_stabilized_MC.png)
 
-## COPTER FLIGHT MODES
-
-## Manual Modes
+#### Acro (Acrobatic Mode)
 
-Manual modes are with the least assistance from the PX4 autopilot or even without assistance to the pilot. 
+For copters this is the 3D flying mode. You can make flips and acrobatics. In acrobatic mode you have fully direct rate control over your aircraft. You have to remember that once you release Pitch and Roll to their center position the craft remains in its current state. You have to give back command in order to level it or change course and direction. In Acrobatic mode you have full rate control over the Roll, Pitch, Yaw and Ttrottle.
 
-**MANUAL - STABILIZED FLIGHT**
+![MC Manual Acrobatic Flight](../../images/flight_modes/manual_acrobatic_MC.png)
 
-In Stabilized flight your Copter will hover in level flight once the Pitch and Roll sticks are at center position. You will have a full manual angle control over the multicopter. If your craft is perfectly set and balanced it will hold the altitude if Throttle is at 50% in the middle of the scale at calm weather. If there is a wind blowing the craft will drift in the direction of the wind and you have to control the throttle all the time during wind gusts to hold altitude. In order to perform a maneuver, you have to hold your Pitch, Roll and Yaw command, otherwise the craft will level itself the moment you release the control sticks.
+#### Rattitude
 
-![MC Manual Flight](../../images/flight_modes/manual_stabilized_MC.png)
+In simple terms it is the mix between Stabilized and Acrobatic mode. So when the sticks are in the center the craft hovers. When you give small movements of the sticks around the center up to the middle the craft will move gently and behave like in stabilize flight (angle control from the center to the middle). When you push the sticks beyond the middle to their extreme positions you have Acrobatic movement i.e. you can perform flips and other figures (rate control at extreme stick positions).
```

Comment:
I haven't "really" updated any of these sections for reasons I will go into at the end. However note that when I do, we will refer to these as "Acro" not Acrobatic mode.

Reply:
I really did not find a words to explain it in better way. Do you know how it works? It is pretty simple, but hard to explain.